src/app/components/Header.tsx

```diff
@@ -43,21 +82,49 @@ const Header = () => {
 
 				{/** keep this div. header will have more items, will be "right-aligned" */}
 				<div>
-					{egoJwt ? (
-						<UserBadge
-							showGreeting={true}
-							firstName={'Test'}
-							lastName={'User'}
-							title={'DCC Member'}
-							css={css`
-								color: white;
-							`}
-						/>
-					) : loggingIn ? (
-						<DnaLoader />
-					) : (
-						<LoginButton />
-					)}
+					<AppBarMenuItem
+						active={profileActive}
+						css={css`
+							border-bottom: ${profileActive ? activeBorder : ''};
+						`}
+					>
+						{egoJwt ? (
+							<FocusWrapper
+								onClick={() => {
+									setDropdownOpen(!isDropdownOpen);
+								}}
+							>
+								{isDropdownOpen && (
+									<DropdownMenu>
+										{profileNavDetails.map((element, idx) => (
+											<NavBarElement
+												key={`profileNavDetail_${idx}`}
+												{...element}
+												isDropdown={true}
+											/>
+										))}
+									</DropdownMenu>
+								)}
+								<UserBadge
+									showGreeting={true}
+									firstName={'Test'}
+									lastName={'User'}
+									title={'DCC Member'}
+									className={onProfilePage ? 'active' : ''}
```

Comment:
this is putting the `active` classname on a div
I don't see related styles that this would apply
```javascript

export const UserBadgeContainer = styled('div')`
  text-align: left;
  display: flex;
  align-items: center;
  justify-content: space-between;
  min-width: 200px;
  margin: 0 -24px;
  padding: 0 16px;
  box-sizing: border-box;
`;
```

Reply:
Removed, looks fine